talk/basicconcepts/classenum.tex

```diff
@@ -1,4 +1,4 @@
-\subsection[Compound]{Compound data types}
+\subsection[Class/Enum]{Class and enum types}
```

Comment:
The term compound type is used wrongly here, because compound types includes e.g. references and pointers. What is meant here are class and enum types. Unions, weirdly, are class types.